src/coordinator/tsdb/remote/compressed_codecs.go

```diff
@@ -226,6 +289,18 @@ func tagIteratorFromCompressedTags(compressedTags []*rpc.CompressedTag, idPool i
 	return tagIter
 }
 
+func tagIteratorFromSeries(series *rpc.Series, iteratorPools encoding.IteratorPools) (ident.TagIterator, error) {
+	compressedValues := series.GetCompressed()
+	if len(compressedValues.GetCompressedTags()) > 0 {
```

Comment:
nil check for compressedValues ? Might be better to check for isSet ?

Reply:
Good call on nil check, for some reason didn't think .GetCompressed() could return nil